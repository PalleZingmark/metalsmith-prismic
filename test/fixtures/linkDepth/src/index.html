---
template: index.hbt
prismic:
  macaron:
    query: '[[:d = at(document.tags, ["Macaron"])] [:d = any(document.type, ["product"])]]'
    orderings: '[my.product.name]'
    linkDepth: 2
---
