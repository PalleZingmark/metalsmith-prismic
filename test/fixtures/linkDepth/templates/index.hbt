<html>
<body>
<ul>
  {{#each prismic.macaron.results}}
    <li>
        {{{data.name.html}}}
        <div class="related">
          {{{data.related.document.data.name.html}}}
          {{{data.related.html}}}
        </div>
        <div class="doubly-related">
          {{{data.related.document.data.related.document.data.name.html}}}
          {{{data.related.document.data.related.html}}}
        </div>
    </li>
  {{/each}}
</ul>
</body>
</html>
