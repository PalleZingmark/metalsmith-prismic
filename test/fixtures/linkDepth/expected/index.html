<html>
<body>
<ul>
    <li>
        <h1>Black &amp; White Macaron</h1>
        <div class="related">
          <h1>Speculoos Macaron</h1>
          <a href="/product/UlfoxUnM0wkXYXbD/speculoos-macaron">/product/UlfoxUnM0wkXYXbD/speculoos-macaron</a>
        </div>
        <div class="doubly-related">
          <h1>Speculoos Cupcake</h1>
          <a href="/product/UlfoxUnM0wkXYXbT/speculoos-cupcake">/product/UlfoxUnM0wkXYXbT/speculoos-cupcake</a>
        </div>
    </li>
    <li>
        <h1>Cool Coconut Macaron</h1>
        <div class="related">
          <h1>Vanilla Macaron</h1>
          <a href="/product/UlfoxUnM0wkXYXbH/vanilla-macaron">/product/UlfoxUnM0wkXYXbH/vanilla-macaron</a>
        </div>
        <div class="doubly-related">
          <h1>Dark Chocolate Macaron</h1>
          <a href="/product/UlfoxUnM0wkXYXbE/dark-chocolate-macaron">/product/UlfoxUnM0wkXYXbE/dark-chocolate-macaron</a>
        </div>
    </li>
    <li>
        <h1>Dark Chocolate Macaron</h1>
        <div class="related">
          <h1>Woodland Cherry Pie</h1>
          <a href="/product/UlfoxUnM0wkXYXbe/woodland-cherry-pie">/product/UlfoxUnM0wkXYXbe/woodland-cherry-pie</a>
        </div>
        <div class="doubly-related">
          <h1>Exotic Kiwi Pie</h1>
          <a href="/product/UlfoxUnM0wkXYXbO/exotic-kiwi-pie">/product/UlfoxUnM0wkXYXbO/exotic-kiwi-pie</a>
        </div>
    </li>
    <li>
        <h1>Pistachio Macaron</h1>
        <div class="related">
          <h1>Vanilla Macaron</h1>
          <a href="/product/UlfoxUnM0wkXYXbH/vanilla-macaron">/product/UlfoxUnM0wkXYXbH/vanilla-macaron</a>
        </div>
        <div class="doubly-related">
          <h1>Dark Chocolate Macaron</h1>
          <a href="/product/UlfoxUnM0wkXYXbE/dark-chocolate-macaron">/product/UlfoxUnM0wkXYXbE/dark-chocolate-macaron</a>
        </div>
    </li>
    <li>
        <h1>Salted Caramel Macaron</h1>
        <div class="related">
          <h1>Dark Chocolate Macaron</h1>
          <a href="/product/UlfoxUnM0wkXYXbE/dark-chocolate-macaron">/product/UlfoxUnM0wkXYXbE/dark-chocolate-macaron</a>
        </div>
        <div class="doubly-related">
          <h1>Woodland Cherry Pie</h1>
          <a href="/product/UlfoxUnM0wkXYXbe/woodland-cherry-pie">/product/UlfoxUnM0wkXYXbe/woodland-cherry-pie</a>
        </div>
    </li>
    <li>
        <h1>Speculoos Macaron</h1>
        <div class="related">
          <h1>Speculoos Cupcake</h1>
          <a href="/product/UlfoxUnM0wkXYXbT/speculoos-cupcake">/product/UlfoxUnM0wkXYXbT/speculoos-cupcake</a>
        </div>
        <div class="doubly-related">
          <h1>Triple Chocolate Cupcake</h1>
          <a href="/product/UlfoxUnM0wkXYXbj/triple-chocolate-cupcake">/product/UlfoxUnM0wkXYXbj/triple-chocolate-cupcake</a>
        </div>
    </li>
    <li>
        <h1>Vanilla Macaron</h1>
        <div class="related">
          <h1>Dark Chocolate Macaron</h1>
          <a href="/product/UlfoxUnM0wkXYXbE/dark-chocolate-macaron">/product/UlfoxUnM0wkXYXbE/dark-chocolate-macaron</a>
        </div>
        <div class="doubly-related">
          <h1>Woodland Cherry Pie</h1>
          <a href="/product/UlfoxUnM0wkXYXbe/woodland-cherry-pie">/product/UlfoxUnM0wkXYXbe/woodland-cherry-pie</a>
        </div>
    </li>
</ul>
</body>
</html>
